dsp-role/core/src/test/scala/dsp/role/api/RoleRepoSpec.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright © 2021 - 2022 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.api
+
+/**
+ * To be implemented...
+ */
+object placeholder {}
```

Comment:
Is this file needed? Are you going to implement it? I don't think we need to test the repo api (it's just an interface). I think it makes more sense to test the implementations of it.

Reply:
Well this I'm not sure. You also have it in the user slice.